packages/contracts/package.json

```diff
@@ -17,13 +17,18 @@
     "standards"
   ],
   "scripts": {
+    "postinstall": "yarn submodules || exit 0",
+    "prebuild": "yarn submodules || exit 0",
```

Comment:
Do we need both of these?

Reply:
We could probably get away with just 1 of them